AnalyticsAdmin/src/V1alpha/AudienceDimensionOrMetricFilter/NumericFilter/Operation.php

```diff
@@ -31,32 +31,18 @@ class Operation
      * Generated from protobuf enum <code>LESS_THAN = 2;</code>
      */
     const LESS_THAN = 2;
-    /**
-     * Less than or equal.
-     *
-     * Generated from protobuf enum <code>LESS_THAN_OR_EQUAL = 3;</code>
-     */
-    const LESS_THAN_OR_EQUAL = 3;
     /**
      * Greater than.
      *
      * Generated from protobuf enum <code>GREATER_THAN = 4;</code>
      */
     const GREATER_THAN = 4;
-    /**
-     * Greater than or equal.
-     *
-     * Generated from protobuf enum <code>GREATER_THAN_OR_EQUAL = 5;</code>
-     */
-    const GREATER_THAN_OR_EQUAL = 5;
```

Comment:
this is a breaking change, but as we are in `preview` (pre-1.0), this is acceptable.